products/dataproc/api.yaml

```diff
@@ -0,0 +1,186 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: Dataproc
+display_name: Cloud Dataproc
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://dataproc.googleapis.com/v1/
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://dataproc.googleapis.com/v1beta2/
+scopes:
+  - https://www.googleapis.com/auth/cloud-identity
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Cloud Dataproc API
+    url: https://cloud.google.com/dataproc/docs/reference/rest/
```

Comment:
I think this field expects a value like `https://console.cloud.google.com/apis/library/dataproc.googleapis.com/`. We don't consume it in Terraform.